index.js

```diff
@@ -62,7 +64,7 @@ function checkVersion (version) {
   }
 
   if (fastifyVersion && !semver.satisfies(fastifyVersion, version)) {
-    throw new Error(`fastify-plugin - expected '${version}' fastify version, '${fastifyVersion}' is installed`)
+    throw new Error(`fastify-plugin: ${pluginName} - expected '${version}' fastify version, '${fastifyVersion}' is installed`)
```

Comment:
Does the `pluginName` have a default it isn't somehow set?

Reply:
The `checkName` function will provide every time a string-name to assign to the plugin, that I have added to `checkVersion()`'s arguments

Reply:
👍